tensilelite/Tensile/Tests/common/gemm/fp16_tn.yaml

```diff
@@ -216,6 +216,8 @@ BenchmarkProblems:
         - BiasTypeArgs: ['h']
         - ActivationArgs:
           - [Enum: relu]
+          - [Enum: silu]
```

Comment:
Should we add "silu" test on ActivationType: hipblaslt_all?

Reply:
Where should I add it?

Reply:
I added silu to gemm_bias_strided_batched.yaml